src/tools/illink/Directory.Build.props

```diff
@@ -0,0 +1,37 @@
+<Project>
```

Comment:
This file should import the repo root's Directory.Build.props file. Same for the Directory.Build.targets file. Otherwise you would need to duplicate a lot properties, i.e. `$(NetCoreAppToolCurrent)`. That makes future changes and versioning tfms harder.

Reply:
Agreed, we need the linker to adopt the arcade infrastructure which is hooked up through these as well.  There are a ton of common settings and build infrastructure that come in through arcade and repo props/targets.  The linker should adopt those.

Reply:
The linker already imports Arcade explicitly in its props and targets files, but we are still missing all the repo infrastructure.

Reply:
I'm fine with doing this in a follow-up PR. There's no problem with merging the repo first, then wiring through the darc infra